store/postgres/migrations/2019-02-26-182914_remove_unassigned_deployment_indexes_procedure/up.sql

```diff
@@ -0,0 +1,41 @@
+-- Find all subgraphs without deployment assignments and remove their indexes
+CREATE OR REPLACE FUNCTION remove_unassigned_deployment_indexes()
+    RETURNS VOID AS
+$$
+DECLARE
+    index_to_drop RECORD;
+    counter INT := 0;
+BEGIN
+    -- Loop through each index
+    FOR index_to_drop IN
+        -- Get all indexes on subgraphs without deployment assignments
+        SELECT
+          indexname as name
+        from pg_indexes
+        where
+          left(indexname,46) IN
```

Comment:
It would be good to have a comment where the magic `46` comes from. I tripped over that reading through it.